invoker/core/src/main/java/com/google/cloud/functions/invoker/Event.java

```diff
@@ -53,6 +55,31 @@ public Event deserialize(
         context =
             jsonDeserializationContext.deserialize(
                 adjustContextResource(contextCopy), CloudFunctionsContext.class);
+      } else if (isPubSubEmulatorPayload(root)) {
+        JsonObject message = root.getAsJsonObject("message");
+
+        String timestampString = message.has("publishTime")
+          ? message.get("publishTime").getAsString()
```

Comment:
Nit: Google style requires continuation lines like this one to be at least +4 relative to the previous line. The same applies to the `.setEventType` stuff further down. Maybe run google-java-format over the files here?

Reply:
I did not realize I could use the google-java-format outside of google3. This is super helpful.